### PR TITLE
[Localization] NEW: Portuguese/Brazil (pt-br)

### PR DESCRIPTION
**Pull request by danilolucasmd**
Hey, I would like to contribute with some localization for us here in Brazil.

### CODE REVIEW

**Comment by IsaacMarovitz** [comment]
Do not merge until the latest Weblate changes have been merged.

File: PlayCover.xcodeproj/project.pbxproj
Comment (Depal1) [inline PlayCover.xcodeproj/project.pbxproj:504]: Is it right to have this "duplicated"? `pt-br` and `pt-BR`.

Comment (danilolucasmd) [reply PlayCover.xcodeproj/project.pbxproj:504]: This was automatically added when I created Localizable.strings inside the pt-br.lproj folder, but I've seen it used both ways here.

**Review by Depal1 — APPROVED** [review]

**Pull request merged**
